src/Illuminate/Queue/Listener.php

```diff
@@ -72,7 +73,7 @@ protected function phpBinary()
      */
     protected function artisanBinary()
     {
-        return defined('ARTISAN_BINARY') ? ARTISAN_BINARY : 'artisan';
+        return artisan_binary();
```

Comment:
```suggestion
        return \Illuminate\Console\Application::artisanBinary();
```

Reply:
This is functionally different since `\Illuminate\Console\Application::artisanBinary()` escapes it using `ProcessUtils::escapeArgument()`